### PR TITLE
ME-89 - Adds partner locations view to partner view

### PR DESCRIPTION
**Pull request by kierangillen**
- https://artsyproduct.atlassian.net/browse/ME-89

#skip_new_tests

### CODE REVIEW

**Comment by kierangillen** [comment]
@ashfurrow what was the question?

**Comment by ashfurrow** [comment]
Oh weird, it didn't post. I think we should remove the `isPad` here: 

https://github.com/artsy/emission/pull/1960/files#diff-ecd513af654a7e2f0ea683f7eb4aacdbR9

File: src/lib/Components/LocationMap/index.tsx [hunk 1]
```diff
@@ -33,6 +33,63 @@ interface Props {
...
+  if (city && postalCode) {
+    return city + ", " + postalCode
+  } else if (city) {
+    return city
+  } else if (postalCode) {
+    return postalCode
+  }
+}
+
+export const tappedOnMap = (address, summary, partnerName, city, postalCode, lat, lng) => {
```
Comment (ds300) [inline src/lib/Components/LocationMap/index.tsx]: Long argument lists like this scare me a little 😅Good candidate for the single arg object pattern I reckon.

File: src/lib/Scenes/Partner/Screens/PartnerLocations.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    relay.loadMore(PAGE_SIZE, error => {
+      if (error) {
+        // FIXME: Handle error
+        console.error("PartnerShows.tsx", error.message)
+      }
+      setFetchingNextPage(false)
+    })
+  }
+
+  console.log("partner", partner)
```
Comment (ds300) [inline src/lib/Scenes/Partner/Screens/PartnerLocations.tsx]: I think this was left in by accident?

File: src/lib/Scenes/Partner/Components/PartnerLocationSection.tsx [hunk 3]
```diff
@@ -1,68 +1,78 @@
...
-    const lastUniqCity = uniqCities.pop()
-    const joinedCities = uniqCities.join(",  ")
-    const locationCountText = `${partner.name} has ${cityLength} locations in`
-    return { locationText: locationCountText, cityText: joinedCities, lastCity: lastUniqCity }
-  }
+}
 
-  const handleSeeAllLocations = () => {
-    // FIXME:
+const createLocationsString = partner => {
```
Comment (ds300) [inline src/lib/Scenes/Partner/Components/PartnerLocationSection.tsx]:
```suggestion
const createLocationsString = (partner: PartnerLocationSection_partner) => {
```

File: src/lib/Scenes/Partner/Components/PartnerShows.tsx [hunk 4]
```diff
@@ -10,38 +10,37 @@ import { PartnerShowsRailContainer as PartnerShowsRail } from "./PartnerShowsRai
 
 const PAGE_SIZE = 6
 
+const ShowGridItem = ({ show, itemIndex }) => {
```
Comment (ds300) [inline src/lib/Scenes/Partner/Components/PartnerShows.tsx]:
```suggestion
const ShowGridItem = ({ show, itemIndex }: {show: PartnerShows_partner['pastShows']['edges']['node'], itemIndex: number}) => {
```

**Comment by artsyit** [comment]
:rocket: PR was released in v1.18.28 :rocket:

**Pull request merged**